docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -875,6 +887,89 @@ To process the received search command on the server side, follow these steps:
 > This feature supports Tizen 5.0 and Higher for Mobile and Wearable.
 
 
+## Updating and Retrieving Abilities
+To update the abilities on the server side, follow these steps:
+
+1. Create the media controller server handle using the `mc_server_create()`:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+   
+2. Set the abilities using the corresponding 'mc_server_set_XXX_ability()', or 'mc_server_set_ability_support()'.
+   The following examples demonstrate setting up various abilities:
```

Comment:
Please change line from 899:

Set the abilities using the corresponding `mc_server_set_XXX_ability()`, or `mc_server_set_ability_support()`.


Also, @nawab-samsung, I'd like to delete "The following examples demonstrate setting up various abilities:" on line 900 and keep current "To update ~".
I don't think this layout is good.
![image](https://user-images.githubusercontent.com/9958374/67089527-cfa34f00-f1e2-11e9-9413-e7d8a62499e6.png)

If we'd like to add "The following examples demonstrate setting up various abilities:" , the layout should be changed a little bit more as like : 

---

Set the abilities using the corresponding 'mc_server_set_XXX_ability()', or 'mc_server_set_ability_support()'.

 The following examples demonstrate setting up various abilities:

   - To update the playback ability,~

   - To update the display mode ability, 
---

**I don't want to do that.**. So please remove added line 900.

Reply:
@mijin85cho 
1. Just added the line to get continuity and the flow. Also to make things more lucid. Moreover, I wished to keep the context of "For example" intact. 

@haejeongkim , Request you to remove line 900

Reply:
I removed "The following examples demonstrate setting up various abilities:"